bin/code_tnhaplotyper2.sh

```
@@ -0,0 +1,50 @@
+gatkResource=$(echo ${gatkResource}| sed 's|dx://||g')
+gatkResource_name=$(dx describe ${gatkResource} --name)
+dx download "$gatkResource" -o "$gatkResource_name"
+tar --no-same-owner -zxvf ${gatkResource_name} --strip 1
+ls
+cd ..
+# give permission to run Sentieon
+chmod -R 777 ${pathToBin}
+# get sentieon license
+source ${pathToBin}/license/license_setup.sh
+set -eu
+export SENTIEON_INSTALL_DIR=${pathToBin}/sentieon-genomics-*
```

Shellcheck (0.10.0)

[warning] Brace expansions and globs are literal in assignments. Quote it or use an array.

(SC2125)